app-config.yaml

```diff
@@ -70,6 +70,8 @@ proxy:
     target: https://api.newrelic.com/v2
     headers:
       X-Api-Key: ${NEW_RELIC_REST_API_KEY}
+    allowedHeaders:
+      - link
```

Comment:
you'll find that these are indented under another key now after the release - you got a bit unlucky that the release went out yesterday and changed the shape of some things slightly